lib/core/utils/publish-metadata.js

```diff
@@ -7,20 +7,32 @@
  * @private
  */
 function getIncompleteReason(checkData, messages) {
+	function getDefaultMsg(messages) {
+		if (messages.incomplete && messages.incomplete.default) {
+			// fall back to the default message if no reason specified
+			return messages.incomplete.default;
+		} else {
+			// TODO: localize it
+			return 'aXe couldn\'t tell the reason. Time to break out the element inspector!';
```

Comment:
This string could be anything. Feel free to suggest alternatives if it's not helpful enough / too cheeky.